### PR TITLE
ISPN-13976 fix indentation in sec roles examples

### PR DESCRIPTION
**Pull request by domiborges**
[ISPN-13976](https://issues.redhat.com/browse/ISPN-13976) 
Needs backport to 13.0.x

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/yaml/authorization_roles_custom.yaml [hunk 1]
```diff
@@ -6,7 +6,9 @@ infinispan:
         roles:
           temprole:
             role:
-              permissions: "ALL_WRITE LISTEN"
+              permissions:
+                - "ALL_WRITE LISTEN"
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/yaml/authorization_roles_custom.yaml:10]: Should be a list:

```yaml
permissions:
  - ALL_WRITE
  - LISTEN
```

File: documentation/src/main/asciidoc/topics/yaml/authorization_roles_custom.yaml [hunk 2]
```diff
@@ -6,7 +6,9 @@ infinispan:
...
           myrole:
             role:
-              permissions: "READ WRITE"
+              permissions:
+                - "READ WRITE"
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/yaml/authorization_roles_custom.yaml:14]: Should be a list

```yaml
permissions:
  - READ
  - WRITE
```

**Comment by dvagnero** [comment]
Thanks @tristantarrant . Should be fixed now

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**